src/scripts/main.js

```diff
@@ -21,7 +21,34 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key1 in food) {
+    const ol = document.createElement('ol');
+
+    ol.textContent = key1;
+    document.body.appendChild(ol);
+
+    if (typeof food[key1] === 'object') {
+      for (const key2 in food[key1]) {
```

Comment:
The `for...in` loop should iterate over `data[key1]` instead of `food[key1]` to utilize the `data` parameter.